app/controllers/my_facilities/drug_stocks_controller.rb

```diff
@@ -77,4 +79,15 @@ def drug_stocks_enabled?
       redirect_to :root
     end
   end
+
+  def populate_facility_sizes
+    @facility_sizes = @accessible_facilities
+      .where(facility_group: @selected_facility_group, zone: @selected_zones)
+      .pluck(:facility_size)
+      .uniq
+      .compact
+      .sort
+      .reject { |size| size == "community" }
+    @facility_sizes = sort_facility_sizes_by_size(@facility_sizes)
+  end
```

Comment:
Where is this used?

Reply:
It overrides the `populate_facility_sizes` method in `MyFacilitiesFiltering`

Reply:
Ah okay, it doesn't very clear to me. Your call though.

Why not do uniq (distinct), compact (where not null), sort (order by), and reject (where not) in sql? Not for performance as much for consistency, and extensibility. Relations are easier to compose in rails than lists :/

Reply:
And yes, if we move this into drug stock query this confusion goes away too.

Reply:
> Ah okay, it doesn't very clear to me. Your call though.
> 
> Why not do uniq (distinct), compact (where not null), sort (order by), and reject (where not) in sql? Not for performance as much for consistency, and extensibility. Relations are easier to compose in rails than lists :/

This is fair, I did it this way to be consistent with how `MyFacilitiesFiltering` does it currently. Can rewrite it in both places I guess. 
This method is really just trying to fetch a list of sizes `["large", "medium", "small", "community"]` that should be displayed in the size drop down, so the user can select a size to filter by. We're excluding community from this list so that the user can't select it in the drop down.

Reply:
> And yes, if we move this into drug stock query this confusion goes away too.

This has nothing to do with the Query, just populates options for the drop down facility size filter.

Reply:
Gotcha. My bad, this is fine then! 👍